lib/explorer/series.ex

```diff
@@ -791,6 +791,26 @@ defmodule Explorer.Series do
 
   def cast(_series, dtype), do: dtype_error("cast/2", dtype, @valid_dtypes)
 
+  @doc """
+  Converts a string to datetime.
+
```

Comment:
We need to document the formats here. Alternatively, we can also link to `Calendar.strftime` in Elixir and mention if there are any differences.